packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -601,6 +601,21 @@ export class CeramicCliUtils {
   }
 }
 
+/**
+ * Helper function: Parse provided port and verify validity or exit process
+ * @param inPort
+ */
+export function _validatePort(inPort) {
```

Comment:
@stbrody @willex Are you ok with using usual names, with no underscores?

Reply:
Agreed, this is an exported function, so no need for an underscore.  I usually use underscores exclusively for private methods within a class to make clear the public vs private API